src/Controller/Component/PrgComponent.php

```diff
@@ -55,9 +55,11 @@ public function conversion($redirect = true)
             }
 
             list($url) = explode('?', $this->request->here(false));
-            return $this->_registry->getController()->redirect(
-                $url . '?' . http_build_query($params)
-            );
+            $queryString = http_build_query($params);
```

Comment:
Even the call to `http_build_query()` can be avoided by ensure `$params` is not empty.